### DISCUSSION TITLE
related table

### DISCUSSION THREAD

**Discussion by kephren226** [whitecube/nova-flexible-content#336] [General]
Hi
i'm trying to add a layout based on a related table 
i have a client model and an account model 
i want to be able to create an account for the client when creating the client itself 
I undersatnd the layout concept 
But how do i save the account ??

**Comment by voidgraphics** [comment]
Use a custom resolver, there is an example in the documentation: https://whitecube.github.io/nova-flexible-content/#/?id=custom-resolver-classes
